lib/sufia/engine.rb

```diff
@@ -1,4 +1,12 @@
 module Sufia
+  class Statefile
+    def self.default
+      return '/tmp/minter-state' if Rails.env.development? || Rails.env.test?
+      raise NotImplementedError, "can't use the default statefile for production" unless Dir.exist?('/var/sufia')
```

Comment:
What if it is in production and I don't have `/var/sufia`, but I've specified an alternative location (lets say `/opt/sufia`).  Won't this raise an error?